app/scripts/util/utils.js

```diff
@@ -33,4 +35,25 @@ function isValidMSISDN(inputtxt){
       return false;
     }
   }
+}
+
+
+
+function beautifyIndent(type, content){
+  
+  if ( type.indexOf('text/xml') >= 0 || type.indexOf('application/xml') >= 0 ){
```

Comment:
I have updated the 'beautifyIndent' function to include these two additional types. I will start thinking about how to make this more flexible using a regular expression